book_options.rb

```diff
@@ -1,29 +1,36 @@
-require './rental_options'
+require 'json'
 
 class BookOptions
   attr_reader :books
 
   def initialize
     @books = []
+
+    return unless File.exist?('./json/book.json')
+
+    file = File.read('./json/book.json')
+    file_data = JSON.parse(file)
+    file_data.each do |book|
+      @books.push(Book.new(book['title'], book['author']))
+    end
+  end
+
+  def save_data
+    File.write('./json/book.json', JSON.dump(@books))
   end
```

Comment:
- [ ] I think not all cases where the files are missing are handled, could you please check on this? I will leave a couple of gifs demonstrating the bugs I encountered.

Creating person:
![bug_create_person](https://user-images.githubusercontent.com/79114233/147084637-b96ac773-d040-4ebf-9a12-9dc0dec4b6d5.gif)

Creating book:
![bug_create_book](https://user-images.githubusercontent.com/79114233/147084632-dc1f3ee9-ab0e-4b21-ae06-5099baeb1f53.gif)

I can't create rentals, since there are no books neither persons.
For the next reviewer, please make sure to verify this.

Reply:
Hi @D4nielJ 👋 ,

Thanks for the feedback